pkg/ipmi/ocp/bootorder.go

```diff
@@ -166,11 +165,11 @@ func CheckBMCBootOrder(i *ipmi.IPMI, bmcBootOverride bool) error {
 			break
 		}
 		if bootType = entry.Booter.TypeName(); len(bootType) > 0 {
-			if bootType == "netboot" {
+			if bootType == "pxeboot" {
```

Comment:
This would replace netboot (fbnetboot) with pxeboot, I think we can keep netboot and add a new case for pxeboot.

Reply:
The problem is that there is only one type of network boot in IPMI boot message.

Reply:
I think we can add one VPD variable 'pxeboot' for selecting pxeboot or fbnetboot. I made a commit for your reference
https://github.com/johnnylinwiwynn/u-root/commit/12555d5192ff0170cac7df518aebeefca7ef629e
@insomniacslk